python/scripts/cirkopt.py

```diff
@@ -100,14 +100,6 @@ def explore(self):
             ),
             default=os.path.join(LIBERATE_DIRECTORY, "netlist_ref/INVX1.sp"),
         )
-        parser.add_argument(
-            "--ldb",
-            help=(
-                "Path to LDB library generated by Liberate. "
-                "Must match settings in Liberate configeration files."
-            ),
-            default=os.path.join(LIBERATE_DIRECTORY, "lib/example_tt_1.0_70_nldm.lib"),
-        )
```

Comment:
It wasn't fully implemented; it would get passed to the parser but wasn't passed around everywhere. It made it seem like you could set it arbitrarily but that wouldn't actually work.